frontend/src/js/components/Store.jsx

```diff
@@ -142,7 +142,12 @@ var Store = React.createClass({
 
   onProceedToPayment: function() {
     this.setState({paymentBegun: true});
-    this.order.preparePayment();
+    var store = this;
```

Comment:
I've been using .bind(this) for these, but this way kinda makes sense too. Still, I think we should maybe stick to one or other.